src/app/app.module.ts

```diff
@@ -193,7 +202,9 @@ export class MyHammerConfig extends HammerGestureConfig {
     AppRoutingModule,
     ChartsModule,
     ScrollDispatchModule,
-    QRCodeModule
+    QRCodeModule,
+    PlatformModule
```

Comment:
מה זה?

Reply:
I am re-adding the QR code
the Paltform Aiham added to support different error message per platform (IOS, Android) for the location service

Reply:
As for the Platform - I'm not sure how it works